web/beacon-app/src/features/auth/components/Register/SuccessfullAccountCreation.tsx

```diff
@@ -1,37 +1,48 @@
 import { Trans } from '@lingui/macro';
 import { Card, Heading } from '@rotational/beacon-core';
+import { Link } from 'react-router-dom';
 
+import { ROUTES } from '@/application';
 import OtterLookingDown from '@/components/icons/otter-looking-down';
 
 function SuccessfullAccountCreation() {
+  const registrationEmail = sessionStorage.getItem('newRegistrationEmail');
+
   return (
     <div className="relative mx-auto mt-20 w-fit pt-20">
       <OtterLookingDown className="absolute -right-16 -top-[10.8rem]" />
       <Card contentClassName="border border-[#72A2C0] rounded-md p-4 md:p-8 text-sm">
         <Card.Header>
           <h1 className="text-[18px] font-bold">
-            <Trans>Thank you for creating your Ensign account!</Trans>
+            <Trans>Let's Go!</Trans>
           </h1>
         </Card.Header>
         <Card.Body>
           <Heading as="h1" className="mt-4 mb-3 ">
             <Trans>
-              We value security and care deeply about protecting your information. For that reason,
-              we request that you verify your email account.{' '}
+              To keep your account safe, we sent a verification email to{' '}
+              {registrationEmail ? (
+                <span className="font-bold">{registrationEmail}</span>
+              ) : (
+                'the email address provided during sign up'
+              )}
+              . Click the secure link in that email to continue.
             </Trans>
           </Heading>
-          <div className="space-y-2">
-            <Heading as="h2" className="text-md font-bold">
-              <Trans>Next Steps</Trans>
-            </Heading>
-            <p>
-              {' '}
-              <Trans>
-                Please check your email and click the secure link in the verification email we just
-                sent you. You can then log into Ensign to start building!
-              </Trans>
-            </p>
-          </div>
+          <p>
+            {' '}
+            <Trans>
+              Didn't receive an email?{' '}
+              <Link to="#" className="font-bold text-[#1F4CED]">
```

Comment:
it should be a button here, the resend will call an action.

Reply:
@masskoder A button here would look a little out of place with the rest of the text, I think. I'll see how it looks with the ghost variant or we may need to only display text that can't be clicked and then add the button below it.